.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
```

Comment:
@rohitcbr what are these three files?

Reply:
@yashgit891 not sure of these files when pushing, should these 3 files be removed?

Reply:
@rohitcbr if they are not part of sdk update and also not made by you then ideally it should not be present.